php/map.jinja

```diff
@@ -64,6 +65,7 @@
         'ldap_pkg': 'php5-ldap',
         'php_ini': '/etc/php5/apache2/php.ini',
         'dev_pkg': 'php5-dev',
+        'dev_pkg': 'php5',
```

Comment:
It's okay, I fixed it in https://github.com/saltstack-formulas/php-formula/pull/106